GitUI/Editor/FileViewer.cs

```diff
@@ -664,7 +672,7 @@ private string GetFileText(string fileName)
             using (var stream = File.Open(path, FileMode.Open, FileAccess.Read, FileShare.ReadWrite))
             using (var reader = new StreamReader(stream, Module.FilesEncoding))
             {
-                var content = reader.ReadToEnd();
+                var content = await reader.ReadToEndAsync().ConfigureAwait(false);
                 FilePreamble = reader.CurrentEncoding.GetPreamble();
```

Comment:
We need to ensure that the continuation can be executed (it can get stale/obsolate (don't know which word is better here) if another file was viewed). Perhaps CancellationTokenSequence.